spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -344,7 +344,8 @@ For convenience, we have the static files with application-URIs (for both maven
 for all the out-of-the-box stream and task/batch app-starters. You can point to this file and import
 all the application-URIs in bulk. Otherwise, as explained previously, you can register them individually or have your own custom property file with only the required application-URIs in it. It is recommended, however, to have a "`focused`" list of desired application-URIs in a custom property file.
 
-The following table lists the bit.ly links to the available Stream Application Starters based on Spring Boot 1.5.x:
+The following table include the bit.ly links to the available Stream Application Starters based on Spring Cloud Stream 1.3.x
```

Comment:
I think a separate section for `Stream App Starters` and `Task App Starters` would help break up the long table listings that make it harder to parse what is going on.